### PR TITLE
Add Docker benchmarking.

### PR DESCRIPTION
**Pull request by spark008**

### CODE REVIEW

File: doc/docker_benchmark.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Usage
+
+To run the benchmark against `JOBS` x 10 Docker containers:
+
+```bash
+rake db:migrate:reset docker:bm JOBS=3
+```
+
+The `docker:bm` task is equivalent to `docker:build_db docker:benchmark`; the
```
Comment (pwnall) [inline doc/docker_benchmark.md]: Can you replace ; by .? I don't see the need to join the two sentences into one.

Comment (pwnall) [inline doc/docker_benchmark.md]: Also, you can say that the task simply invokes the `docker:build_db` and `docker:benchmark` tasks in order. You don't have to specify the rake commands for executing the individual tasks, most developers should be able to assemble them on their own.

**Comment by pwnall** [comment]
LGTM. Please merge after fixing the doc comments.

Thank you!

**Comment by spark008** [comment]
I'm not sure if I interpreted your comments correctly, so I just deleted the explanations of the task breakdown. Is that close to what you meant?

**Pull request merged**